### ISSUE TITLE
cfnspec missing

### ISSUE THREAD

**Issue by chick-fil-a-21** [awslabs/aws-cdk#576]
9:/# npm install -g aws-cdk
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @aws-cdk/cfnspec@^0.8.2
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget 
npm ERR! notarget It was specified as a dependency of '@aws-cdk/cloudformation-diff'
npm ERR! notarget 

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-08-15T14_33_13_122Z-debug.log

**Comment by rix0rrr** [comment]
Hi @chick-fil-a-21,

I wonder if this is a case of eventual consistency; we just published the new `0.8.2`, and when I try to install `@aws-cdk/cfnspec 0.8.2` on my machine, it does seem to work:

```shell
$ npm install -g @aws-cdk/cfnspec
+ @aws-cdk/cfnspec@0.8.2
added 5 packages from 5 contributors in 1.908s
```

I can also see the [0.8.2 version up on npmjs.com](https://www.npmjs.com/package/@aws-cdk/cfnspec).

Does it work if you retry?

**Comment by rix0rrr** [comment]
Closing because I assume you were installing while we were updating. Please reopen if there are still issues.
